src/entity_resolver/scripts.py

```diff
@@ -32,6 +34,7 @@ def create_subparser(subparsers):
 def _norm_arxiv(input_path, output_path):
     with open(output_path, 'w') as f:
         f.write('arxiv')
+    pass
```

Comment:
Remove `pass`

Reply:
Although the two files seem to be able to share the same processing function, I think a better approach would be to write the shared function and call it inside both `_norm_arxiv` and `_norm_citeseer`.